Block/Adminhtml/Timeline/Index.php

```diff
@@ -55,36 +56,39 @@ class Index extends Template
      */
     private $config;
 
+    /**
+     * @var CronDataService
+     */
+    private $cronDataService;
+
     public function __construct(
         Context $context,
-        Collection $collection,
         DateTime $dateTime,
         DateTimeConverter $converter,
         CronConfiguration $config,
+        CronDataService $cronDataService,
         array $data = []
     ) {
         $this->dateTime = $dateTime;
         $this->cronData = [];
         $this->converter = $converter;
         $this->config = $config;
+        $this->cronDataService = $cronDataService;
 
-        $this->initializeStartAndEndTime($collection);
+        $this->initializeStartAndEndTime();
         parent::__construct($context, $data);
     }
 
-    private function initializeStartAndEndTime(Collection $scheduleCollection): void
+    private function initializeStartAndEndTime(): void
     {
         $minDate = null;
         $maxDate = null;
 
         /** @var Schedule $schedule */
```

Comment:
```
assert($schedule instanceof Schedule)
```

Reply:
According the coding standards that Magento 2 provider, we're not allow to use the `assert` function. Though I do agree this is a proper use-case for it.
To be compliant with the coding standards of Magento, I've replaced the assert with an if statement.